guides/v2.2/architecture/archi_perspectives/arch_diagrams.md

```diff
@@ -12,7 +12,7 @@ Depending upon your role and purpose for learning more about Magento, there are
 
 The following diagram illustrates the components and shows the "layers" or tiers in Magento.
 
-![]({{site.baseurl}}/common/images/archi_diagrams_current.png)
+![]({{site.baseurl}}/common/images/archi_diagram_desired-state.png)
```

Comment:
`desired-state` indicates that this is not what the current Magento architecture looks like. Can this be noted on the page to prevent confusion.

Also, I'm not sure this is a 2.2 change?

Reply:
GraphQL is 2.3 only, and Query API doesn't exist as of 2.3.3.